packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -46,7 +46,7 @@ function validationIssuesToLspDiagnostics(
     };
 
     const issueKind = currIssue.kind;
-    switch (issueKind) {
+    switch (currIssue.kind) {
```

Comment:
Can you remove the casting now in line 68?

Reply:
No, this change above was mistakenly committed(WIP), either the assetNever does not work or casting is needed. So as discussed previously we choose **assertNever** as it provides the bigger value.

Reply:
It works in `tooltip.ts`. I wonder what's different there.